src/avram/base_query_template.cr

```diff
@@ -25,7 +25,7 @@ class Avram::BaseQueryTemplate
       macro generate_criteria_method(query_class, name, type)
         def \{{ name }}
           column_name = "#{@@table_name}.\{{ name }}"
-          \{{ type }}::Lucky::Criteria(\{{ query_class }}, \{{ type }}).new(self, "#{@@table_name}.\{{ name }}")
+          \{{ type }}::Lucky::Criteria(\{{ query_class }}, \{{ type }}).new(self, column_name)
```

Comment:
The `column_name` variable was never actually being used. We could also just get rid of it too

Reply:
I like that it uses the name so it's clear what it is. Good catch on actually using it though haha